web/src/assets/styles/blocks.scss

```diff
@@ -275,15 +275,58 @@ span.notification-mark[data-variant="sidebar"] {
     display: grid;
 
     gap: var(--spacer-small);
-    grid-template-columns: 1fr 2fr 2fr;
-    grid-template-areas:
-      "type-and-size drive-info drive-content"
-    ;
-
-    > :first-child {
-      align-self: center;
-      text-align: center;
-      justify-self: start;
+
+    grid-template-columns: 1fr;
+    grid-template-areas: "content";
+    text-align: start;
+
+    > [data-type="details"] {
+      font-size: 80%;
+    }
+
+    &:hover {
+      &:not([aria-selected]) {
+        background: var(--color-gray-dark);
+      }
+    }
+
+    &:is([data-type="locale"]) {
```

Comment:
JFTR, I think I badly explained what I wanted to achieve with the data-type attribute in these selectors. My idea was more about ending up having something similar to the HTML <input>. I.e., adding such an attribute to the selector itself, not to the options.

In any case, please keep it as it is now. We know we have to rework selectors and this is something can be re-evaluated an addressed when doing so.